buildSrc/build.gradle

```diff
@@ -0,0 +1,26 @@
+plugins {
+    id 'java-gradle-plugin'
+    id "com.diffplug.spotless" version "6.13.0"
```

Comment:
Small nit:
```suggestion
    id 'com.diffplug.spotless' version '6.13.0'
```

Reply:
Thank you. 
I revised it in https://github.com/scalar-labs/scalardb/pull/1749/commits/6869be5cc5673c960281f044475812ca45db1a88